src/test/java/reactor/netty/SocketUtils.java

```diff
@@ -355,6 +355,7 @@ else if(maxPort > PORT_RANGE_MAX){
 		 * @throws IllegalStateException
 		 * 		if the requested number of available ports could not be found
 		 */
+		@SuppressWarnings("JdkObsolete")
```

Comment:
consider simply replacing the type of the `availablePorts` variable below from `SortedSet` to `TreeSet`, to limit the amount of suppresswarnings

Reply:
fixed